JobConfig/ensemble/scripts/Stage2_submitensemble.sh

```diff
@@ -52,20 +52,19 @@ done
 NJOBS="" #to help calculate the number of events per job
 LIVETIME="" #seconds
 RUN=1201
-DIO_EMIN=""
+DIO_EMIN=95
 RPC_EMIN=""
```

Comment:
Will there ever be a case where EMIN is different for different components? It would seem hard to analyze in that case. If these will always be the same I suggest a global EMIN value.

Reply:
they are different